src/js/operations/Geo.js

```diff
@@ -0,0 +1,33 @@
+L.DNC.Geo = L.Class.extend({
```

Comment:
Ah, okay, I see where you're coming from.

Now that the execution is broken out from the operation, it seems like whether it's a `turf` operation or another type of operation is not as relevant.  Where it will be relevant is with the execution. I imagine that we'll sooner or later have to add an `executor` property to each `Geo` operation that points to different subclasses of `GeoExecute` or something.  

It's probably not worth focusing too much on this now, as we don't know which other geo-operations will be added in the future or how they'll need to be implemented.